scripts/qa_crowbarsetup.sh

```diff
@@ -5038,10 +5039,147 @@ function ping_fips
     done
 }
 
+function get_stack_fips
```

Comment:
why not make it a "proper" function and return/print the results instead of saving them to a file here? the caller can grab them and save to file or use directly.

Reply:
Files are easier to handly in shell, since...